lib/mutant/mutator/node/named_value/access.rb

```diff
@@ -19,6 +19,44 @@ def dispatch
             emit_singletons
           end
 
+          # Named value access emitter for instance variables
+          class Ivar < Access
+
+            handle(:ivar)
+
+            children :name
+
+            # Emit mutations
+            #
+            # @return [undefined]
+            #
+            # @api private
+            def dispatch
+              emit_attribute
+              super()
+            end
+
+            private
+
+            # Emit instance variable as attribute send
+            #
+            # @return [undefined]
+            #
+            # @api private
+            def emit_attribute
```

Comment:
The name of the method is very generic. Lets name it something like `emit_attribute_read`?